addons/mail/static/src/components/attachment_link_preview/attachment_link_preview.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AttachmentLinkPreview" owl="1">
+        <div class="o_AttachmentLinkPreview rounded px-3 pt-3 pb-2 d-flex" t-attf-class="{{ className }}" t-ref="root">
+            <t t-if="attachmentLinkPreviewView and attachmentLinkPreviewView.attachment">
+                <img t-if="attachmentLinkPreviewView.attachment.mimetype === 'application/o-linkpreview-with-thumbnail'" class="o_AttachmentLinkPreview_linkImage rounded mr-3 mb-2" t-att-src="attachmentLinkPreviewView.imageUrl" t-att-alt="attachmentLinkPreviewView.attachment.name"/>
+                <div t-if="attachmentLinkPreviewView.attachment.name" class="o_AttachmentLinkPreview_content">
+                    <a t-att-href="attachmentLinkPreviewView.attachment.url" class="o_AttachmentLinkPreview_title">
```

Comment:
Haven't tested, but I suppose clicking on link go to URL in place of current Odoo page.
I think the intended behaviour would be to open a new tab even by just a click.
Reason for not liking changing page is because it leaves the Odoo web app, and because Odoo is a web app that still takes some time to load, it's quite annoying to come back. Also the back button of browser doesn't (fully) restore previous state of Odoo web app, which is another annoyance.

Reply:
It should always be the user choice to open a new tab or not. We are now forcing them, and it's sad.

Reply:
I don't think so.
Sometimes choice is bothersome, especially when the default behaviour is basically stupid and annoying, which for me is the case here by just relying on default behavior of `<a>` (i.e. open page in place).

Reply:
You are forcing a specific behavior on the user. They don't have a choice here.